### ISSUE TITLE
Love and use your app daily! Improvements?

### ISSUE THREAD

**Issue by phakeEHS** [dupontgu/simple-cast#2]
I loved your Medium post and using your barebones Android app everyday since! Thanks so much!

Did you see that aschober updated his Vinyl Cast app?
https://github.com/aschober/vinyl-cast/releases/tag/v1.1.0

Any thoughts on free or 3rd party services for audio based ACR that could be baked in?

I love your app because I can launch it from my phone and not have it tied to a device by the digital converter.

**Comment by dupontgu** [comment]
So glad you're finding it useful!  
I had no idea they were still working on it, to be honest.
I definitely thought about adding some ACR, but it was pretty low on my priority list.  I don't have any display hooked up, so initially I didn't know what I would do with that info once I had it.  Also, I figured that, in my home scenario, I always knew what was playing because either my wife or I physically put it on.  However, I looove the idea of being able to scrobble or otherwise broadcast what I'm listening to to the outside world.  I'll take a look when I get a chance.  I think I'd have more options since the code would run on the raspi instead of an Android device. Let me know if you have any specific feature requests.

**Comment by phakeEHS** [comment]
Yes man, it's next level rad. I love it. Since I have a Google Mini's around the house + a Nest Hub Max display + an older Nest Hub display it would be rad to see that artwork or even a track. No pressure though. Might even use Chromecast Ultra to my All Speaker group for added radness.

Agreed since it's running on the Linux there should be a more robust scalability. 🤞

I'll look into it as well.
